spiped/spiped.1

```diff
@@ -43,12 +43,17 @@ spiped - secure pipe daemon
 .SH OPTIONS
 .TP
 .B \-e
-Take unencrypted connections from the source socket and send
-encrypted connections to the target socket.
+Take unencrypted connections from the
+.I source socket
+and send
```

Comment:
Forgot to remove the line break.

Assuming your review is finished (last message was half an hour ago): I just fixed the issues.